### ISSUE TITLE
Is it possible to visualize the loss graph during fine-tuning from Mistral to Zephyr?

### ISSUE THREAD

**Issue by sooftware** [huggingface/alignment-handbook#10]
I really appreciate you sharing this valuable project. 
Is there a way to view the loss graph when fine-tuning from Mistral to Zephyr? This could be a valuable resource for research.

**Comment by sebastianschramm** [comment]
When you are running any of the zephyr-7b-beta recipes (and you did not change the default values in the yaml config files for `report_to`), it will log all metrics to tensorboard. 
And you can visualize the tensorboard logs during (and after) training by running `tensorboard --logdir <RUNS_FOLDER_INSIDE_OF_OTUPTU_DIR>`. Just make sure to use the path to the `runs` directory inside of the output_dir (see the config yaml file) as logdir. For example, if you run everything with the provided default values it should be `tensorboard --logir data/zephyr-7b-sft-lora/runs` for the LoRA SFT step.

**Comment by sooftware** [comment]
Thank you.
